Wikipedia/Code/AbuseFilterAlertView.swift

```diff
@@ -2,67 +2,66 @@
+            subtitleLabel.text = nil
+            detailsLabel1.text = WMFLocalizedStringWithDefaultValue("abuse-filter-disallow-unconstructive", nil, nil, "An automated filter has identified this edit as potentially unconstructive or as a vandalism attempt. Please go back and change your edit.", "Label text for unconstructive edit description")
+            detailsLabel2.text = nil
+            detailsLabel3.text = nil
+            detailsLabel4.text = nil
+            detailsLabel5.text = nil
         case .warning:
             titleLabel.text = WMFLocalizedStringWithDefaultValue("abuse-filter-warning-heading", nil, nil, "This looks like an unconstructive edit, are you sure you want to publish it?", "Header text for unconstructive edit warning")
-            subtitle1Label.text = WMFLocalizedStringWithDefaultValue("abuse-filter-warning-subheading", nil, nil, "Your edit may contain one or more of the following:", "Subheading text for potentially unconstructive edit warning")
-            subtitle2Label.text = WMFLocalizedStringWithDefaultValue("abuse-filter-warning-caps", nil, nil, "Typing in ALL CAPS", "Label text for typing in all capitals")
-            subtitle3Label.text = WMFLocalizedStringWithDefaultValue("abuse-filter-warning-blanking", nil, nil, "Blanking articles or spamming", "Label text for blanking articles or spamming")
-            subtitle4Label.text = WMFLocalizedStringWithDefaultValue("abuse-filter-warning-irrelevant", nil, nil, "Irrelevant external links or images", "Label text for irrelevant external links and images")
-            subtitle5Label.text = WMFLocalizedStringWithDefaultValue("abuse-filter-warning-repeat", nil, nil, "Repeeeeating characters", "Label text for repeating characters")
+            subtitleLabel.text = WMFLocalizedStringWithDefaultValue("abuse-filter-warning-subheading", nil, nil, "Your edit may contain:", "Subheading text for potentially unconstructive edit warning")
+            detailsLabel1.text = WMFLocalizedStringWithDefaultValue("abuse-filter-warning-caps", nil, nil, "All caps text", "Label text for typing in all capitals")
+            detailsLabel2.text = WMFLocalizedStringWithDefaultValue("abuse-filter-warning-blanking", nil, nil, "Deleting sections or full articles", "Label text for blanking sections or articles")
+            detailsLabel3.text = WMFLocalizedStringWithDefaultValue("abuse-filter-warning-spam", nil, nil, "Adding spam to articles", "Label text for adding spam to articles")
+            detailsLabel4.text = WMFLocalizedStringWithDefaultValue("abuse-filter-warning-irrelevant", nil, nil, "Irrelevant external links or images", "Label text for irrelevant external links and images")
+            detailsLabel5.text = WMFLocalizedStringWithDefaultValue("abuse-filter-warning-repeat", nil, nil, "Repeating characters", "Label text for repeating characters")
```

Comment:
^ all instances of `WMFLocalizedStringWithDefaultValue` should use the swift version `WMFLocalizedString(_, value:, comment:`

Reply:
gah oops will fix...

Reply:
fixed!